src/api/types.ts

```diff
@@ -224,7 +224,7 @@ export interface HostCreateParams {
   hostId: string; // uuid
 }
 export type HostList = Host[];
-export interface Introspection {
+export interface HardwareInfo {
```

Comment:
This file is autogenerated from `src/api/swagger.json` using `yarn swagger` script. So we should not change it.